### ISSUE TITLE
Delete Key from Server using Private Key

### ISSUE THREAD

**Issue by tcyrus** [keys-pub/keys-ext#64]

**Comment by gabriel** [comment]
Yeah we could add a delete request to the web api to remove a sigchain from the server. It would probably remove the entries but keep a record of the key id, so that the key couldn't be re-used (or statements republished)?
